services/painting-process-equipment-defect-detection-model-service/tests/test_predict.py

```
@@ -0,0 +1,203 @@
+import pytest
+from fastapi import FastAPI, status
+from fastapi.testclient import TestClient
+from unittest.mock import patch, MagicMock, call
```

Ruff (0.12.2)

`unittest.mock.call` imported but unused

Remove unused import: `unittest.mock.call`

(F401)